swap_meet/decor.py

```diff
@@ -1,2 +1,12 @@
-class Decor:
-    pass
\ No newline at end of file
+from .item import Item
+class Decor(Item):
+    def __init__(self, id = None, width = 0, length = 0, condition = 0, age = None):
+        super().__init__(id, condition, age)
+        self.width = width
+        self.length = length
+    def get_category(self):
+        return "Decor"
```

Comment:
See comment above in `Clothing` and below in `Item` about how to remove this method (avoids unnecessary overriding and extra code) so that you can inherit this behavior from the parent class.